portal/templates/gil/about.html

```diff
@@ -76,22 +78,22 @@ <h4 class="icon-box__head">Care Plan</h4><span class="icon-box__lowhead">Navigat
           <p class="module__heading">Our USA Partners</p>
           <p class="module__body">We have brought together a Network that is actively working with us on the best tools for living with and beyond prostate cancer.</p>
           <div class="partner-grid">
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/colorado.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/DanaFarber.png') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/duke.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/emory.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/johnshopkins.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/karmanos.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/sloan.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/michigan.gif') }}"></div>
-            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/moffitt.gif') }}"></div>
+            <div class="partner-grid__item"><img src="{{url_for('static', filename='img/colorado.gif') }}" alt="University of Colorado Cancer Center"></div>
```

Comment:
yeah, Paul, you may have noticed I didn't put tranlation tags on GIL pages, I plan to do them en mass in a separate PR as most GIL pages don't have them just yet.